### PR TITLE
Change navbar background to yellow

### PR DESCRIPTION
**Pull request by trouni**
## Scope of this PR

Changed the navbar color to yellow.

RailsAuthentication

### CODE REVIEW

**Comment by dmbf29** [comment]
interesting choice in color. who came up with this?

**Comment by dmbf29** [comment]
probably Jonas

**Comment by trouni** [comment]
Thanks @dmbf29, this is Jonas' responsibility.

**Review by dmbf29 — APPROVED** [review]
LGTM 👍

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .navbar-lewagon {
   justify-content: space-between;
-  background: white;
+  background: yellow;
```
Comment (trouni) [inline app/assets/stylesheets/components/_navbar.scss:3]: Yeah, I'm not too sure about this, but let's give it a shot.

**Pull request merged**